src/components/popUp/popUpSrv.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+(function () {
+    angular.module('znk.infra.popUp').factory('PopUpSrv',[
+        '$injector', '$q', '$rootScope', '$animate',
+        function ($injector, $q, $rootScope, $animate) {
+            var PopUpSrv = {};
+
+            var $body = angular.element(document.querySelector('body'));
+            $body.append('<div class="znk-popup"></div>');
+            var popUpsPlaceHolderElement = angular.element(document.querySelector('.znk-popup'));
```

Comment:
maybe you can avoid this selector....
http://stackoverflow.com/questions/1443233/easier-way-to-get-a-jquery-object-from-appended-element